src/main/java/org/sonar/plugins/jacoco/XmlReportParser.java

```diff
@@ -105,6 +105,18 @@ private static String getStringAttr(XMLStreamReader parser, String name, Supplie
     return value;
   }
 
+  private static int getOptionalIntAttr(XMLStreamReader parser, String name, Supplier<String> errorContext) {
+    String value = parser.getAttributeValue(null, name);
+    if (value == null) {
+      return 0;
+    }
+    try {
```

Comment:
A potential improvement could be to extract the logic inside this `try-catch` to a utility method, as it is the same logic as the `try-catch` inside `getIntAttr`